Measurements/Tasks.qs

```diff
@@ -134,8 +134,9 @@ namespace Quantum.Kata.Measurements {
     //      1) N qubits which are guaranteed to be
     //         in one of the two superposition states described by the given arrays of bit strings.
     //      2) two arrays of bit strings represented as Bool[][]s.
-    //         Each of the arrays has a size M ⨯ N and consists of M bit strings of length N
-    //         (the first index is the index of the bit string, the second - the index of the qubit).
+    //         The arrays have dimensions M₁ ⨯ N and M₂ ⨯ N respectively, where N is the number of
+    // 	       qubits and M₁ and M₂ can be different (the first index is the index of
```

Comment:
I'd say "and M₁ and M₂ are the numbers of bit strings in each array. Note that in general M₁ ≠ M₂" - this way we explain what they are, not just that they are different.

Reply:
Done